cicd/gitops/run-gitops-control-plane-workload-on-infra-nodes.adoc

```diff
@@ -6,9 +6,13 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can use infrastructure nodes to isolate infrastructure workloads for two primary purposes:
+Infrastructure nodes prevent additional billing cost against subscription counts. 
 
-* Prevent incurring billing costs against subscription counts.
-* Separate maintenance and management.
+{product-title} allows certain workloads installed by the {gitops-title} Operator to run on Infrastructure nodes. This comprises the workloads that are installed by the {gitops-title} Operator by default in the `openshift-gitops` namespace, including the default Argo CD instance in that namespace.
```

Comment:
IBM SG: [Allow:] Avoid stating that inanimate objects grant abilities to people, as in "the product allows you to...". Whenever possible, use a direct, user-focused alternative such as "you can use the product to..." or "with this product, users can..."

Reply:
Most instances of "infrastructure node" in openshift-docs are not capitalized. (...also "worker node" and "compute node")

Reply:
Suggestion: "specific workloads".

Reply:
Suggestion (active voice): "These include workloads that the {gitops-title} Operator installs by default in the openshift-gitops namespace, including the default Argo CD instance in that namespace."